controllers/snapshot/snapshot_adapter_test.go

```diff
@@ -699,9 +738,12 @@ var _ = Describe("Snapshot Adapter", Ordered, func() {
 			components := []applicationapiv1alpha1.Component{
 				*hasComp,
 			}
-			snapshotEnvironmentBinding, err := adapter.createSnapshotEnvironmentBindingForSnapshot(adapter.application, env, hasSnapshot, &components)
+			temp_labels := map[string]string{}
```

Comment:
nit: `temp_labels` would probably be easier to understand as something like `newLabels`.

Reply:
Will updated